engine/uk.co.spudsoft.birt.emitters.excel/src/uk/co/spudsoft/birt/emitters/excel/StyleManagerHUtils.java

```diff
@@ -197,6 +197,12 @@ public void applyBorderStyle(Workbook workbook, CellStyle style, BorderSide side
 							// log.debug( "Bottom border: " + xStyle.getBorderBottom() + " / " +
 							// xStyle.getBottomBorderXSSFColor().getARGBHex() );
 							break;
+						case DIAGONAL:

```

Comment:
@speckyspooky 

This needs attention.  There are new enumerators and I have no idea the correct handling for these case.